### ISSUE TITLE
REST API

### ISSUE THREAD

**Issue by APT-ZERO** [Leiren/Npanel#24]
Hello
It's great if you create an API to create/modify/remove users and else
Thank you

**Comment by Leiren** [comment]
yes , now mode has this also, but im not sure if im going to enable it right now or next version(s)
it needs some ui design and docs to be written
